### ISSUE TITLE
[TSX] Missing trailing comma on arrow functions using markup

### ISSUE THREAD

**Issue by santiagodoldan** [palantir/tslint#3511]
### Bug Report

- __TSLint version__: 5.8.0
- __TypeScript version__: 2.6.1
- __Running TSLint via__: nvim + ale

#### TypeScript code being linted

```ts
  public render() {
    return (
      <div>
        {
          [1, 2, 3].map((n) =>
            <span>
              { n }
            </span> <===== here requires a comma
          )
        }
      </div>
    )
  }

```

with `tslint.json` configuration:

```json
{
  "awesomeTypescriptLoaderOptions": {
    "errorsAsWarnings": true
  },
  "compilerOptions": {
    "module": "commonjs",
    "target": "es5",
    "lib": ["es2016", "dom"],
    "noImplicitAny": true,
    "noUnusedLocals": true,
    "strictNullChecks": false,
    "sourceMap": true,
    "outDir": "dist",
    "jsx": "react",
    "baseUrl": ".",
    "experimentalDecorators": true,
    "emitDecoratorMetadata": true
  },
  "include": [
    "./src/**/*"
  ]
}

```

#### Actual behavior
It requires an extra comma when maybe not needed.

#### Expected behavior
As how it works TSX it could be a good idea to not have to add the extra comma when adding markup in a single line arrow function like above. Saying that I know I can do this and it works but ...

```
        {
          [1, 2, 3].map((n) => {
            return (
              <span>
                { n }
              </span>
            )
          })
        }
```

... it would be better to avoid all those lines if possible.

**Comment by ajafff** [comment]
Duplicate of #2872

See https://github.com/palantir/tslint/issues/2872#issuecomment-306418677 for more details.

**Comment by santiagodoldan** [comment]
Yep, I saw that other issue, but I was hoping that this could be different only in .tsx files. Anyway thanks 👍
